db/migrate/20211005125938_remove_columns_from_newbooking.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveColumnsFromNewbooking < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :newbookings, :date, :string
```

Comment:
something to be aware of here...

if you have bookings in the system already, when you run this you will lose all of the dates and times for all of the bookings.

A different pattern for this is:
```ruby
# create a new class locally to this migration with the name being unique
# this is to stop any future changes breaking things...
class NewBookings20211005125938 < ApplicationRecord
  # tell the class that it uses the newbookings table
  self.table_name = 'newbookings' 
end

def up
  # create the new column
  add_column :newbookings, :session, :datetime

  # migrate the data from the old columns to the new column
  NewBookings20211005125938.each do |booking|
    # figure out what the date and time should be
    new_session = Time.parse("#{booking.date} #{booking.time}")

    # update the record
    booking.update(session: new_session)
  end

  # now we have the new_session with the data, we can  drop the old columns
  remove_column :newbookings, :date, :string
  remove_column :newbookings, :time, :string
end
```